addons/web_editor/static/src/js/editor/odoo-editor/src/commands/deleteForward.js

```diff
@@ -163,7 +163,13 @@ HTMLElement.prototype.oDeleteForward = function (offset) {
         return;
     }
 
-    const nextSibling = this.nextSibling;
+    let nextSibling = this.nextSibling;
+    while (nextSibling && isWhitespace(nextSibling)) {
+        // If the next sibling is a whitespace, remove it.
+        nextSibling.remove();
+        nextSibling = this.nextSibling;
+    }
+
```

Comment:
I'm kind of confused as to why this change might fix the bug? I mean the change only affects whitespace but somehow fixes the behavior around contenteditables? How? ^^'

Reply:
The issue produce when the html structure is like: `<p><br>[]</p>\n<div contentEditable="false"></div>`
In the deleteForward, when we find nextsibling of this using `this.nextSibling` it will be a whitespace. As a result, the code that remove `contentEditable=false` elements does not execute, as the next sibling is not a `contentEditable=false` element.

so removing that whitespce before the code that remove `contentEditable=false` elements is executed fix the issue.

Reply:
That seems too broad a fix, I mean I fear we might be deleting some whitespace that has useful value at some point in the future in some edge case due to that code. Could you maybe add a check that the whitespace has a cType of BLOCK_OUTSIDE on both sides? I think that would ensure that it is in fact not rendered. I'm not sure so let me know if it works.

Reply:
@dmo-odoo i have applied the requested changes and its working